packages/common/src/dotlottie-player.ts

```diff
@@ -986,11 +982,12 @@ export class DotLottiePlayer {
     let shouldRender = false;
 
     if (revertOptions.includes('activeAnimationId')) {
-      const activeAnimationId = this._manifest?.activeAnimationId;
+      const activeAnimationId = this._getActiveAnimationId();
```

Comment:
This situation could be a concern. The main purpose of `revertToManifestValues` is to reset specific settings to how they were originally defined in the Manifest. 

For instance, if a user has an `activeAnimationId` in the player props, would the this._getActiveAnimationId(); function retrieve the `activeAnimationId` from player props?

However, what we really need here is to get the `activeAnimationId` from the Manifest, even if the user has set their own `activeAnimationId`.

Reply:
Suggestion: Adding the following to above this line
```
this._activeAnimationId = this._dotLottieLoader.manifest.activeAnimationId;
```

Reply:
@afsalz Nice catch!! 👍🏼